examples/next/pages/ui/components/geo/use-map-hook/index.page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, MapView } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import { useMap } from 'react-map-gl';
+
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+
+Amplify.configure(awsExports);
+
+function FlyToButton() {
+  const { current: map } = useMap();
+
+  const flyToMordor = () => {
+    map.flyTo({ center: [172.78, -42.28], zoom: 5 });
+  };
+
+  return <Button onClick={flyToMordor}>Fly, you fools!</Button>;
```

Comment:
Not a native English speaker...so just to confirm...is it good to write `fools` here? 😂

Reply:
Since a Balrog is a foe beyond any of you, this wording is appropriate.